AlbumDownloader.py

```diff
@@ -41,7 +47,11 @@ def find_album_songs_wiki(album_title, artist, google_songs=[]):
     res = requests.get(search, headers=HEADERS_GET).text
     soup = BeautifulSoup(res, 'html.parser')
 
-    first_result_link = soup.find(name='div', attrs=GOOGLE_SEARCH_RESULTS_ATTRS).findChild(name='a').get(key='href')
+    try:
+        first_result_link = soup.find(name='div', attrs=GOOGLE_SEARCH_RESULTS_ATTRS).findChild(name='a').get(key='href')
+    except AttributeError:
+        print("we didn't find anything that match what you wrote")
+        return
```

Comment:
add the album title and the artist to the print.
also, I would have print it in the main, after the "if None" check.
if you dont want to that's ok